my-pern-app/backend/src/server.ts

```diff
@@ -44,10 +44,10 @@ app.put('/api/updateMovie/:movieId', async (req, res, next)=>{
         const {newTitle, newSummary, newImdbLink, newRating} = req.body
         const sql = `
         update "movies"
-        set "title" = $2
-        set "summary" = $3
-        set "imdbLink" = $5
```

Comment:
Lines 11-14:
You should only be creating one pool. The second approach is how we did it in class, but we put the connection string in an environment variable, named DATABASE_URL. I strongly recommend using an environment variable rather than hard-coding the value.

You probably also need to add the field: 
```
ssl: {
    rejectUnauthorized: false,
  }
```

Reply:
what is the purpose of the field: 
ssl: {
    rejectUnauthorized: false,
  }

?

Reply:
When i add this field, I am unable to make httpie requests from the terminal it seems. I get this error:
Error: The server does not support SSL connections
    at /Users/andrewowyang/Code/movieBackend/my-pern-app/backend/node_modules/pg-pool/index.js:45:11
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
Error: The server does not support SSL connections
    at /Users/andrewowyang/Code/movieBackend/my-pern-app/backend/node_modules/pg-pool/index.js:45:11
    at processTicksAndRejections (node:internal/process/task_queues:95:5)

when i take this field out, my httpie request from the terminal works, I tried this with a get request from my terminal: http get http://localhost:5001/api/getMovies

Reply:
`rejectUnauthorized: false` tells Postgres to allow non-SSL traffic. It is required in some environments, but actually causes errors in other environments! So you need to just play with it until you get a setting that works. It looks like with your setup, you don't want it.